wp/lib/bap_wp/src/symbol.ml

```diff
@@ -26,7 +26,7 @@ let build_command (filename : string) : string =
   Format.sprintf "objdump -t %s | sed '0,/SYMBOL TABLE:/d' | grep -E 'data|bss' | awk '{print $1, $NF}'" filename
 
 let run_command (cmd : string) : string list =
-  let inp = Unix.open_process_in cmd in
+  let inp = Caml_unix.open_process_in cmd in
   let r = In_channel.input_lines inp in
   In_channel.close inp; r
```

Comment:
If `open_process_in` throws an exception, you don't catch that, so you'll have a channel that's not properly closed when the program terminates. That isn't a big deal, unless you're spawning lots of processes, or you don't ever reboot your machine.

This isn't something to fix now, but maybe a ticket should be opened for it?

(Good examples of ensuring channels are closed can be found in the source code for the standard `Unix` module's `open_process*` functions, or in the `scram` module.)